Makefile

```diff
@@ -158,13 +158,14 @@ help:
 	@echo 'OPTIONS:'
 	@echo ''
 	@echo '  FLAVORS - ceph-container images to operate on in the form'
-	@echo '    <CEPH_VERSION>,<BASEOS_REPO>,<BASEOS_TAG>'
+	@echo '    <CEPH_VERSION>[CEPH_POINT_RELEASE],<BASEOS_REPO>,<BASEOS_TAG>'
 	@echo '    and multiple forms may be separated by spaces.'
 	@echo '      CEPH_VERSION - named ceph version (e.g., luminous, mimic)'
-	@echo '      BASEOS_REPO - The base image to use for the daemon-base container. This is also'
-	@echo '                    the distro path sourced from ceph-container (e.g., ubuntu, centos)'
-	@echo '      BASEOS_TAG - Tagged version of the base repo to use. Also the distro version'
-	@echo '                   sourced from ceph-container (e.g., ubuntu:"16.04", centos:"7")'
+	@echo '      CEPH_POINT_RELEASE - Optional field to select a particular version if Ceph (e.g -12.2.2)'
```

Comment:
I would add more clarification that dash '-' and equal '=' are the supported point release separators and which one to use should be based on the package manager used in the image. Dash for yum/zypper. Equal for apt.

Reply:
Added.